R/write-plumber.R

```diff
@@ -12,6 +12,8 @@
 #' working directory. See [plumber::plumb()] for naming precedence rules.
 #' @param rsconnect Create a Plumber file with features needed for [Posit
 #' Connect](https://posit.co/products/enterprise/connect/)? Defaults to `TRUE`.
+#' @param additional_pkgs Any additional R packages that need to be **attached**
```

Comment:
I called this `additional_pkgs` just like the argument in `vetiver_write_docker()` 🤪 because I thought it would be worse to give it a different name, even though they are different kinds of extra packages needed (one set needs to be just installed and one set needs to be fully attached). @isabelizimm do you have a different opinion here?

Reply:
There is no way to accidentally send what you mean to use for one arg to the other one, but maybe this is too confusing for people to understand easily. I could make them `additional_docker_pkgs` and `additional_plumber_pkgs`, for more clarity. Or `additional_installed_pkgs` and `additional_attached_pkgs`. This would involve deprecating the existing `additional_pkgs` argument.

Reply:
I think `additional_pkgs` seems fine-- I don't think we want to try to differentiate between docker/plumber/etc packages, since you will be installing all of them at the endpoint regardless, right?